src/inc/msquic.hpp

```diff
@@ -805,6 +806,15 @@ struct MsQuicConnection {
         MsQuic->ConnectionShutdown(Handle, Flags, ErrorCode);
     }
 
+    void
+    Close()
+    {
```

Comment:
```suggestion
```suggestion
    void
    Close(
        ) noexcept {
```
```